google/cloud/spanner_dbapi/parse_utils.py

```diff
@@ -174,12 +177,11 @@
 RE_PYFORMAT = re.compile(r"(%s|%\([^\(\)]+\)s)+", re.DOTALL)
 
 
+@deprecated(reason="This method is deprecated. Use _classify_stmt method")
```

Comment:
this pulls in two new dependencies (deprecated, wrapt) into this library -- this can be replaced entirely with this inside the function body:

```python
warnings.warn("This method is deprecated.  Use classify_stmt method instead", stacklevel=2)
```

this uses `warnings` directly avoiding the two new dependencies (including one which has a C extension!)

Reply:
would one be open to a PR to do this instead?

Reply:
@ankiaga Would you mind taking a look what would be the best solution here?

Reply:
threw one together: https://github.com/googleapis/python-spanner/pull/1120